image/scripts/app_launch

```diff
@@ -1,19 +1,11 @@
 #!/bin/bash
 
-#Ensure background processes are killed when the script ends, otherwise pebble
-# will never see the process ending and think it's still running
-trap 'trap - SIGTERM && kill 0' SIGINT SIGTERM EXIT
-
-cd "${CONTAINER_APP_ROOT}/app" || exit 1
-
 export UNICORN_BIND_ALL=0.0.0.0
 export UNICORN_SIDEKIQS=1
 
-# Putting the tail's in the background gets the various logs outputting to
-# stdout. The sed allows us to distinguish where a given log line comes form.
-tail -f "${CONTAINER_APP_ROOT}/app/log/production.log" | sed 's/^/discourse: /' &
-tail -f "${CONTAINER_APP_ROOT}/app/log/unicorn-stderr.log" | sed 's/^/unicorn-error: /' 1>&2 &
+su -s /bin/bash -c "${CONTAINER_APP_ROOT}/app/bin/bundle exec prometheus_exporter" &
```

Comment:
The log won't be redirected anymore to stdout?

Reply:
No, it will be collected by loki